spec/bowling_game_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'bowling_game'
```

Comment:
your tests were really clear and thorough. I like the context blocks. Your use of doubles has also made it really clear to follow!